source/server/server.cc

```diff
@@ -825,6 +827,9 @@ void InstanceImpl::terminate() {
   // Before the workers start exiting we should disable stat threading.
   stats_store_.shutdownThreading();
 
+  Config::GrpcMuxImpl::shutdownAll();
+  Config::NewGrpcMuxImpl::shutdownAll();
```

Comment:
Yeah this really doesn't thrill me. Can we have a TODO here and also a tracking issue for cleaning this up? I get we need to get this done and get this shipped but it would be good to track better the crash and what we can do to avoid the statics here.

Reply:
Yeah, it doesn't thrill me either. Added a TODO.